Add Timestamps to the startup printout and fetch messages

craigpoma opened the pull request:
This PR will add a new output to the startup message that will tell you the "Software Launch Time". It will also add time stamps to the INFO lines for when it does a fetch from Bricklink or BrickOwl.

This was added because the standard output did not show the fetch times - so - if the process had locked up (which happens periodically) - you could never really tell when it last made a pull - nor could you easily identify it was locked up.

The software version has been bumped to: 1.7.4-1701389201 where the last number is the seconds from epoch when the build occurred. I saw the last builds were using the DATE STAMP (i.e. Nov 10, 2023 thus the build was 231110) so you can swap the build number to 231130 - if that make more sense on release.

Example output with the new fields highlight in yellow as call outs:
![image](https://github.com/ZZJHONS/Bricksync/assets/26184942/4c850846-951a-49e4-b0f0-ea53332c95be)

craigpoma commented:
For user that might want this before it is merged and built into the ZZJHONS release, I have a release published for windows located in my GitHub that I PR'ed the feature over from. The release is only built for WIN64 and can be pulled from: https://github.com/craigpoma/Bricksync/releases/tag/1.7.4-1701545059

ZZJHONS commented:
Thanks for your contribution

ZZJHONS commented:
We will have to take a look at this again, for me the startup time is always showing the current time when we input the STATUS command.

craigpoma commented:
@ZZJHONS - Yes - I see the issue. I added the "Launch Time" line to the bsCommandStatus function (in bricksyncinput.c) in this PR. That function gets called at startup time (line 1177 of Bricksync.c). Unfortunately, where I added the two lines of code that control that output - didn't account for a user calling "status" at a later point :-O - wasn't thinking about that :-/. The start time should have been stored off in a persistent value after first calculated at startup (in Bricksync.c) and then been passed into the bsCommandStatus versus calculated there.  I can look at send a patch over to you this weekend.

craigpoma commented:
@ZZJHONS - See https://github.com/ZZJHONS/Bricksync/issues/44 and PR https://github.com/ZZJHONS/Bricksync/pull/43 - Sorry for the inadvertent issue.
